package.json

```diff
@@ -35,6 +35,8 @@
     "exports": false
   },
   "devDependencies": {
+    "chai": "^3.5.0",
```

Comment:
Yeah. I was previously using the matches assertions but I removed it.
On 17 May 2016 1:13 AM, "Hugo Giraudel" <notifications@github.com> wrote:

> In package.json
> <https://github.com/kaelig/sass-project-boilerplate/pull/1#discussion_r63369726>
> :
>
> > @@ -35,6 +35,8 @@
> >      "exports": false
> >    },
> >    "devDependencies": {
> > +    "chai": "^3.5.0",
>
> I think the original assert library from Node.js is enough without having
> to introduce Chai.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaelig/sass-project-boilerplate/pull/1/files/4c6935f6f309b5617eadc1d7c7f49d23f64a9ede#r63369726>
>